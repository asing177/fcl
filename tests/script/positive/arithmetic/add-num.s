decimal<2> m = 0.01;
num      n = 1/3;

@initial
init() {
  n = m + n; // expect n = 103/300
  transitionTo(@terminal)
}
