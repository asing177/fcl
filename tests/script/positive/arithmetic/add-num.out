 
Method arguments and effects:
init: () -> {read m, read n, write n}